.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,16 @@
 
 Hey!
 
-Type: bug fix | new feature | refactoring | code quality | documentation
+Type: bug fix | new feature | code quality | documentation
 Link to issue:
```

Comment:
Seems we need 1 space line
https://github.com/ddmler/phpsa/blob/4cc055d3e3e2b3672e77d109429664a8176caf44/.github/PULL_REQUEST_TEMPLATE.md
To split this blocks